### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by grantland**
Adds `docker-compose` instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +64,28 @@ docker create \
     cpoppema/docker-flexget
 ```
 
+### docker-compose
+
+```
+---
+  flexget:
```
Comment (cpoppema) [inline README.md:73]: If you can turn this into a fully working yml with the following bit that'd be great :smile:

```
version: "2"

services:
  flexget:
    .....
```

Comment (grantland) [reply README.md:73]: done :)

**Comment by grantland** [comment]
Updated :)

**Comment by cpoppema** [comment]
Thank you :smile:

**Pull request merged**